engine/src/main/java/org/terasology/engine/logic/characters/CharacterSoundSystem.java

```diff
@@ -60,7 +60,7 @@ public void onFootstep(FootstepEvent event, EntityRef entity, LocationComponent
         Block block = worldProvider.getBlock(blockPos);
         if (block != null) {
             if (block.getSounds() == null) {
-                logger.error("Block '{}' has no sounds", block.getURI());
+                logger.atError().addArgument(() -> block.getURI()).log("Block '{}' has no sounds");
```

Comment:
Simple getter for member access
```suggestion
                logger.error("Block '{}' has no sounds", block.getURI()); //NOPMD
```